packages/djangoflow_error_reporters/djangoflow_sentry_reporter/example/lib/main.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:djangoflow_error_reporter/djangoflow_error_reporter.dart';
```

Comment:
This is not what we discussed, it should not DjangoflowSentryReporter
I wrote the psuedo code please not them or ask me to provide them if I forget. It should be 
class ExampleErrorReporter extends ErrorReporter.... with implemented methods that will just debugPrint data. Very simple and straightforward.